XiaoMi/pegasus
# refactor: refactor the code about prometheus in pegasus_counter_reporter

### What problem does this PR solve? 
refactor the code in pegasus_counter_reporter about prometheus.

### What is changed and how it works?
We used pushgateway before, but it is strongly recommended not to use now. So we use the pull method by prometheus server.

- Manual test (add detailed scripts or steps below)
1.start onebox
```
./run.sh start_onebox -c
```
2.start prometheus server
```
cd {dir of prometheus}
./prometheus --config.file=prometheus.yml
```
3.start grafana
```
cd {dir of grafana}
./bin/grafana-server web
```
4.open grafana by localhost:3000, to see if we can get pegasus dashboard or not.

Related changes
- Need to cherry-pick to the release branch
